### ISSUE TITLE
Discussion of OOs issues etc.

### ISSUE THREAD

**Issue by ldcouto** [overturetool/language#34]
This thread is for following up the discussion from https://github.com/overturetool/overture/issues/493

To summarise grotesquely... OO semantics are lacking. What is to be done about it and what is the role of the LB in this.

**Comment by nickbattle** [comment]
And, I suppose, how does this relate to the current efforts to specify semantics in UTP.

**Comment by ldcouto** [comment]
Right... we should try to figure out which issues are "solved" by the semantics, as delivered and which ones need further work. To me, this ties into the role of the LB. I said I would like it to be a bit more proactive rather than just responding to RMs. Marcel suggested I should just create more RMs but I'd like something more. I'd like us to set and track some goals for each term. 

But maybe I'm wrong and the sole task of the LB should be to move RMs along. But then that raises the question... who drives the evolution of the language?

To focus the thread, maybe we should table the discussions about VDMTools until we know what exactly is going to happen with it.

**Comment by kgpierce** [comment]
Hi, is this an appropriate place to gather questions for Simon, or should we start a new issue?

**Comment by ldcouto** [comment]
Sure. We can discuss the questions here but should probably collate them on
a wiki page.

**Comment by nickbattle** [comment]
See also issue #57.
